src/content/getting-started/designers/designers.md

```diff
@@ -47,17 +45,16 @@ The Carbon Design System team often makes improvements to the Design Kit. We tra
 	- To update to the latest Carbon library version you will have to download the newest version of the Carbon Design Kit each time it is released and replace the previous version.
 	- Follow the specific [instructions](https://github.com/IBM/carbon-design-kit/wiki/Sketch-Libraries-Overview) on the wiki to ensure a smooth transition.
 
-## Use with Box Drive
+### Sync via Box Drive
```

Comment:
Not sure we should say "Sync" since the old Box application is `Box Sync` which we don't want people to use. Maybe "connect"?

Reply:
Changed to "Synchronize" to avoid any possible confusion.